_posts/2024-10-16-18.0.0-release.md

```diff
@@ -0,0 +1,242 @@
+you to the [complete changelog][3].
+
+## Community
+
+Since the 17.0.0 release, JJJJJ has been invited to be committer.
+No new members have joined the Project Management Committee (PMC).
+
+Thanks for your contributions and participation in the project!
+
+## Columnar format
+
+The Arrow columnar format now allows 32-bit and 64-bit decimal data, in
+addition to the already existing 128-bit and 256-bit decimal data types
+(GH-43956).
+
+## Linux packages notes
+
+Azure file system is enabled.
+
+## C Data Interface notes
+
+
+## Arrow Flight RPC notes
+
+**Flight UCX is deprecated.** We plan to remove this experiment in the next couple of releases.
+
+The Java implementation now transparently handles compressed Arrow data when reading, instead of requiring explicit configuration. (GH-43469)
+
+The Ruby bindings now support implementing DoPut on the server. (GH-43814)
+
+## C++ notes
+
+The default memory pool has changed to mimalloc on all platforms (GH-43254).
+Previously, jemalloc was used by default on Linux. Using mimalloc by default
+provides a more consistent experience accross different platforms, and
+makes configuration easier. It is expected that this might either increase
+or decrease performance on user workloads that use the default memory pool;
+please benchmark accordingly. Jemalloc can still be selected by setting
+the `ARROW_DEFAULT_MEMORY_POOL` environment variable to "jemalloc".
+
+A new class `arrow::ArrayStatistics` has been added to encode basic statistics
+about an Arrow array. It provides a source-agnostic representation for statistics
+provided by third-party sources such as Parquet files (GH-41909).
+
+The new Decimal32 and Decimal64 types have been made available (GH-43956).
+
+Several canonical extension types have been implemented:
+- the [Opaque](https://arrow.apache.org/docs/dev/format/CanonicalExtensions.html#opaque) extension type (GH-43454);
+- the [8-bit boolean](https://arrow.apache.org/docs/dev/format/CanonicalExtensions.html#bit-boolean) extension type (GH-17682);
+- the [UUID](https://arrow.apache.org/docs/dev/format/CanonicalExtensions.html#uuid) extension type (GH-15058);
+- the [JSON](https://arrow.apache.org/docs/dev/format/CanonicalExtensions.html#json) extension type (GH-32538).
+
+### Acero
+
+- Enhanced the row-oriented representation by widening the offset type from 32-bit to 64-bit, resolving crashes and data corruption in aggregation and hash join on large datasets due to offset overflow (GH-43495).
+- Improved ordered aggregation performance by reducing complexity from `O(n*m)` to `O(n)`, where `n` is the number of rows and `m` the number of segments in the batch (GH-44052).
+
+### Compute
+
+Casting between string-like and string-view-like types has been implemented (GH-42247).
+
+### Dataset
+
+
+### Filesystems
+
+Writing small files to S3 now uses a single S3 API call instead of three
+(GH-40557). Files larger than 5 MB still go through the regular multipart
+upload mechanism.
```

Comment:
```suggestion
Writing small files to S3 can use a single S3 API call instead of three,
provided the new option `allow_delayed_open` is enabled (GH-40557).
Files larger than 5 MB still go through the regular multipart
upload mechanism.
```